.travis.yml

```diff
@@ -61,8 +63,9 @@ deploy:
 
   # add release_v*.zip file to tagged Releases on github
   - provider: releases
-    api_key: $GITHUB_GNOSIS_INFO_API_TOKEN
-    file: release_$TRAVIS_TAG.zip
+    api_key: $GITHUB_RELEASE_TAG_API_TOKEN
+    body: "IPFS hash: $IPFS_HASH"
```

Comment:
Can we make it a file too the hash? 

This way, anyone could:
- Verify for example that an ENS domain matches a IPFS hash
- Automate getting the IPFS web address
- Automate updating a ENS record easily